pkg/agent/rollouts.go

```diff
@@ -12,16 +12,22 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+// rolloutAction indicates the action to be performed on a Rollout object
 type rolloutAction string
 
 const (
-	rolloutActionPause  = rolloutAction("pause")
+	// rolloutActionPause represents the "pause" action on a Rollout
+	rolloutActionPause = rolloutAction("pause")
+	// rolloutActionResume represents the "resume" action on a Rollout
 	rolloutActionResume = rolloutAction("resume")
-	rolloutActionAbort  = rolloutAction("abort")
+	// rolloutActionAbort represents the "abort" action on a Rollout
+	rolloutActionAbort = rolloutAction("abort")
 )
 
+// rolloutsGetterFactory is a factory for creating RolloutsGetter.
```

Comment:
So what's the purpose of a `RolloutsGetter`?

Reply:
The purpose of `RolloutsGetter` is to fetch Rollouts in a namespace inside the cluster.